lib/src/wip/autocomplete/popup_controller.dart

```diff
@@ -23,6 +23,10 @@ class PopupController extends ChangeNotifier {
 
   int get selectedIndex => _selectedIndex;
 
+  void reset() {
+    itemScrollController = ItemScrollController();
```

Comment:
What is the desired effect of this? Scroll to the top? If so, can you manually scroll to the top?

Replacing the controller has the drawbacks:
- Need to dispose the old one.
- Need to make sure the disposed one will not show in a widget and break it.